pulumi/python/kubernetes/observability/otel-operator/opentelemetry-operator.yaml

```diff
@@ -61,6 +61,131 @@ spec:
             description: InstrumentationSpec defines the desired state of OpenTelemetry
               SDK and instrumentation.
             properties:
+              dotnet:
```

Comment:
What is the relation of this change to the version bumps that are the meat of this PR? Is this extra configuration required by a new version of one of the libraries?

Reply:
This is strictly a copy of the yaml manifest for OTEL - https://opentelemetry.io/docs/k8s-operator/

Why do we include it here instead of just reaching out to pull it down each time? Paranoia....I don't want this to change and blow up MARA for someone. I want to control the versions, especially since cert-manager and OTEL seem to be super touchy about each other.